test/kaffe/group_member/worker/worker_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Kaffe.WorkerTest do
+
+  use ExUnit.Case
+
+  alias Kaffe.Worker
+  
+  defmodule TestListHandler do
+    def handle_messages(messages) do
+      Enum.each messages, fn (m) ->
+        send :test_case, {:handle_messages, m.message}
+        :ok
+      end
+    end
+  end
+
+  defmodule TestSingleHandler do
+    def handle_message(message) do
+      send :test_case, {:handle_message, message.message}
+      :ok
+    end
+  end
+
+  test "use list handler" do
+    Process.register(self(), :test_case)
+    {:ok, worker_pid} = Worker.start_link(TestListHandler, 0)
+    Worker.process_messages(worker_pid, [%{message: :one}, %{message: :two}])
+
+    :timer.sleep 100
+
+    assert_received {:handle_messages, :one}
```

Comment:
Gotcha. I'm unsure if I have enough context to assess, but I was thinking that one thing you could be testing is that multiple messages get sent at once.